frontend/server/src/Controllers/Course.php

```diff
@@ -3002,7 +3005,7 @@ public static function apiAddTeachingAssistant(\OmegaUp\Request $r): array {
         ];
     }
 
-    /**
+     /**
```

Comment:
```suggestion
    /**
```

esto estaba bien antes.